.jscsrc

```diff
@@ -1,5 +1,5 @@
 {
 	"preset": "jquery",
-
+	"maximumLineLength": 110,
```

Comment:
I tried searching the style guide for it, but my find fu was poor I suppose...

So if it was "defined correctly" why wasn't it erroring on my long lines?